packages/server/src/koaJsonRpc/index.ts

```diff
@@ -27,104 +27,83 @@ import path from 'path';
 import { Logger } from 'pino';
 
 import {
-  ParseError,
-  InvalidRequest,
   InternalError,
+  InvalidRequest,
   IPRateLimitExceeded,
-  MethodNotFound,
-  Unauthorized,
   JsonRpcError as JsonRpcErrorServer,
+  MethodNotFound,
+  ParseError,
 } from './lib/RpcError';
 import Koa from 'koa';
 import { Histogram, Registry } from 'prom-client';
 import { JsonRpcError, predefined } from '@hashgraph/json-rpc-relay';
-import { RpcErrorCodeToStatusMap, HttpStatusCodeAndMessage } from './lib/HttpStatusCodeAndMessage';
+import { RpcErrorCodeToStatusMap } from './lib/HttpStatusCodeAndMessage';
+import {
+  getBatchRequestsEnabled,
+  getBatchRequestsMaxSize,
+  getDefaultRateLimit,
+  getLimitDuration,
+  getRequestIdIsOptional,
+  hasOwnProperty,
+} from './lib/utils';
+import { IJsonRpcRequest } from './lib/IJsonRpcRequest';
 
-const hasOwnProperty = (obj, prop) => Object.prototype.hasOwnProperty.call(obj, prop);
 dotenv.config({ path: path.resolve(__dirname, '../../../../../.env') });
 
-import constants from '@hashgraph/json-rpc-relay/dist/lib/constants';
-
-const INTERNAL_ERROR = 'INTERNAL ERROR';
-const INVALID_PARAMS_ERROR = 'INVALID PARAMS ERROR';
 const INVALID_REQUEST = 'INVALID REQUEST';
-const IP_RATE_LIMIT_EXCEEDED = 'IP RATE LIMIT EXCEEDED';
-const JSON_RPC_ERROR = 'JSON RPC ERROR';
-const CONTRACT_REVERT = 'CONTRACT REVERT';
-const METHOD_NOT_FOUND = 'METHOD NOT FOUND';
 const REQUEST_ID_HEADER_NAME = 'X-Request-Id';
 const responseSuccessStatusCode = '200';
-const BATCH_REQUEST_METHOD_NAME = 'batch_request';
+const METRIC_HISTOGRAM_NAME = 'rpc_relay_method_result';
```

Comment:
The `const BATCH_REQUEST_METHOD_NAME = 'batch_request';` variable was used only for the rate limiting logic which got removed and since it was not used anywhere else I removed the declaration.

As for the `const METRIC_HISTOGRAM_NAME = 'rpc_relay_method_result'`, it was previously a scoped variable inside the constructor but I moved it to a global variable inside the file for better overall visibility. 

Thus, no new changes have been introduced to the metric dashboards.

Reply:
Thanks